src/main/resources/db/migration/V81__add_mitatoity_and_suostumus_peruttu_timestamps.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE poistettu_opiskeluoikeus
+    ADD COLUMN IF NOT EXISTS mitatoity TIMESTAMP,
+    ADD COLUMN IF NOT EXISTS suostumus_peruttu TIMESTAMP;
+
+UPDATE poistettu_opiskeluoikeus SET suostumus_peruttu = aikaleima;
```

Comment:
Voisikohan tämän hoitaa `ALTER TABLE poistettu_opiskeluoikeus RENAME COLUMN aikaleima TO suostumus_peruttu`?

Reply:
Korjattu tämä siten, että aikaleima-sarake pudotetaan pois ja rivit järjestetään ajan mukaan haettaessa uusien sarakkeiden perusteella.